File: 프로그래머스/0/120889. 삼각형의 완성조건 （1）/삼각형의 완성조건 （1）.js
function solution(sides) {
    let answer = 0;
    sides.sort(function(a, b) {
        return a - b;
    });
    if (sides[0] + sides[1] <= sides[2]) {
        answer = 2;
    }
    else
        answer = 1;
    return answer;
}

File: 프로그래머스/0/120889. 삼각형의 완성조건 （1）/삼각형의 완성조건 （1）.kt
import java.util.Arrays;

class Solution {
    fun solution(sides: IntArray): Int {
        var answer: Int = 0
        
        Arrays.sort(sides);
        
        if (sides[0] + sides[1] <= sides[2]) {
            answer = 2
        }
        else
            answer = 1
        
        return answer
    }
}

// class Solution {
//     fun solution(sides: IntArray): Int {
//         var sides = sides.sorted()
//         return if( sides[2] < sides[0]+sides[1] ) 1 else 2
//     }
// }
